jx_sqlite/insert_table.py

```diff
@@ -281,7 +282,29 @@ def _flatten(data, uid, parent_id, order, full_path, nested_path, row=None, guid
                 elif c.type=="nested" and value_type=="object":
                     value_type = "nested"
                     v = [v]
-
+                elif c.es_index!=table:
```

Comment:
Trying to change doc collection rather than accessing the database for changes. The first approach from deepops1(the pull request that i created for two different approaches: deepops1 and 2). Data is inserted in a proper manner.